ratis-client/src/main/java/org/apache/ratis/client/RaftClient.java

```diff
@@ -64,6 +65,9 @@
   /** Get the {@link SnapshotManagementApi} for the given server. */
   SnapshotManagementApi getSnapshotManagementApi(RaftPeerId server);
 
+  /** Get the {@link LeaderElectionManagementApi} for the given server. */
+  LeaderElectionManagementApi leaderElectionManagementApi(RaftPeerId server);
```

Comment:
Let's call this getLeaderElectionManagementApi since the other method names start with "get".

Reply:
done